hbook/ntup/ntup_example_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup_test
+
+import (
+	"fmt"
+	"log"
+
+	"go-hep.org/x/hep/hbook/ntup/ntcsv"
+)
+
+func ExampleNtuple_scanH2D() {
+	nt, err := ntcsv.Open(
+		"testdata/simple-with-header.csv",
```

Comment:
you could use directly the file from `ntcsv/testdata`:

```go
nt, err := ntcsv.Open(
 	"ntcsv/testdata/simple-with-header.csv",
	ntcsv.Comma(';'),
	ntcsv.Header(),
	ntcsv.Columns("v1", "v2", "v3"),
)
```

so we don't duplicate too many files :)

Reply:
I am getting `open ntcsv/testdata/simple-with-header.csv: no such file or directory
exit status 1
`

Reply:
I think, because ntup_example_test.go is not in the ntup package. That's why this is happening.

Reply:
hum... worked for me.
also, from `ExampleNtuple_scanH2D` point of view, `testdata/...` isn't either in the `ntup` package :)

Reply:
@sbinet, I think ntcsv should be outside ntup, something like this 
 https://github.com/jaybatra26/hep/commit/49ade4a87ffee544cf626ed5c0e4becf0a000ad4

In this commit, I moved the example file to ntcsv. WDYT?

Reply:
umm, why didn't it work for me? Maybe because of  wrong `GOPATH` and `GOROOT` :(

Yep, I realized it as soon as I wrote :)

Reply:
My env variables are as follows. 
`GOPATH=/home/jbatra/go/
GOROOT=/usr/local/go
` 
and the project is in `/home/jbatra/go/src/go-hep.org/` which technically lies in `GOPATH`. ANy idea what could have gone wrong?

Reply:
Woooah!! It worked.

Reply:
cool. what did you change?
if I am not mistaken, one shouldn't set `$GOROOT` if it already points to the location `go` expects (IIRC, `/usr/local/go`)

Reply:
I changes nothing. Just reset the branch. Any way it worked :)